docs/modules/ROOT/pages/explanations/policies.adoc

```diff
@@ -0,0 +1,314 @@
+====
+
+=== Policies which are executed for all requests by unprivileged principals
+
+The following policies are executed for all namespaces which are created by an unprivileged principal:
+
+xref:references/policies/02_disallow_reserved_namespaces.adoc[`disallow-reserved-namespaces` (labeled `RSVD`)]::
+This policy ensures that users can't create namespaces which match a pattern which is reserved for the system.
+We need this policy to ensure that users can't adversely impact the system by using namespace names which might be used by the system in the future.
++
+Effectively, this is a restriction of the requirement that https://kb.vshn.ch/appuio-cloud/references/quality-requirements/usability/ns-arbitrary-name.html[users can choose arbitrary namespace names] to ensure overall system availability.
+The component allows operators to configure the set of disallowed patterns.
+
+xref:references/policies/12_namespace_quota_per_zone.adoc[`check-namespace-quota` (labeled `NS_QUOT`)]::
+This policy denies creation of new namespaces for an organization which has used up their namespace quota on a zone.
+The component allows operators to adjust the global and per-organization namespace quota.
++
+This policy implements the requirement that a single organization can only https://kb.vshn.ch/appuio-cloud/references/quality-requirements/performance/ns-quota.html[create a limited number of namespaces] on an APPUiO Cloud zone.
+
+xref:references/policies/02_validate_namespace_metadata.adoc[`validate-namespace-metadata` (labeled `NS_LBL`)]::
+This policy ensures that users can only create or edit selected labels and annotations.
+In particular, we want to allow users to transfer namespace ownership between organizations of which they're members.
++
+As shown in the https://kb.vshn.ch/appuio-cloud/references/quality-requirements/usability/ns-organization-rbac.html[end-user documentation], transferring a namespace is done by changing the namespace's `appuio.io/organization` label to the organization which should receive ownership of the namespace.
++
+To protect the cluster, we need to ensure that users can't modify arbitrary labels or annotations on a namespaces, since OpenShift exposes a number of privileged operations (such as setting a namespace-wide node selector for workloads) as labels and annotations on namespace objects.
++
+TIP: This policy is executed when namespaces are created or updated.
+
+xref:references/policies/10_generate_default_rolebinding_in_ns.adoc[`default-rolebinding-in-ns` (labeled `ORG_RBAC`)]::
+This policy is applied to any namespace which is created with an organization label.
+It grants role `admin` to the organization to which the new namespace belongs.
++
+This fulfils the requirement that https://kb.vshn.ch/appuio-cloud/references/quality-requirements/usability/ns-organization-rbac.html[namespaces are owned by organizations].
+
```

Comment:
We never talk about project templates. That we create a rolebinding for the creating user and patch it afterwards. 

It's a minor details, so it might not make sense to mention it in the overview, but it definitely is a special layer of duct tape that we IMO should point out somewhere

Reply:
It's already pointed out in the policy reference documentation, cf. https://github.com/appuio/component-appuio-cloud/blob/master/docs/modules/ROOT/pages/references/policies/10_generate_default_rolebinding_in_ns.adoc and I now added a TIP box in the overview.